### ISSUE TITLE
Keyboard shortcuts for Play/Pause should be allowed to be the same

### ISSUE THREAD

**Issue by rpspringuel** [frescobaldi/frescobaldi#1149]
When using the midi interface, the "Media Play" and "Media Pause" commands should be allowed to live on the same shortcut because this is effectively a toggle setting.  "Media Play" has no effect if midi is already playing and "Media Pause" has no effect if midi is not currently playing.

**Comment by uliska** [comment]
I don't think this is possible *directly*. Without having looked into the source code I'd assume that these two should be merged into *one* "Play/Pause" action that will then *always* have only one shortcut. Whenever that is triggered it should update itself to display the "other" action.
